src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -503,6 +503,12 @@
   - name: --snapshot-id
     type: string
     short-summary: The source snapshot id used to create this cluster.
+  - name: --kubelet-config
+    type: string
+    short-summary: Kubelet configurations for agent nodes.
```

Comment:
I am not sure if it's better to remind the user that this should be the path to a json format configuration file.

Reply:
Also a link indicates what properties/options the configuration file can have?